service/share/testing.go

```diff
@@ -242,3 +250,7 @@ func NewBrokenAvailability() Availability {
 func (b *brokenAvailability) SharesAvailable(context.Context, *Root) error {
 	return ErrNotAvailable
 }
+
+func Null() *routing.RoutingDiscovery {
```

Comment:
`share.Nill()` returning RoutingDiscoverry looks off. WDYT about making the below instead
```go
func TestLightAvailability(t *testing.T)
``` 

which would create mock blockservice and nil routing inside, so that Daser tests could use it directly.